webapp/src/Controller/Jury/RejudgingController.php

```diff
@@ -685,4 +693,155 @@ private function generateFlashMessagesForSkippedJudgings(array $skipped): void
             $this->addFlash('danger', $msg);
         }
     }
+
+    private function getStats(Rejudging $rejudging)
+    {
+        $judgings = $this->em->createQueryBuilder()
+            ->from(Judging::class, 'j')
+            ->leftJoin('j.runs', 'jr', 'j.judgingid = jr.judgingid')
+            ->leftJoin('j.rejudging', 'r', 'rejudging.rejudgingid = j.rejudgingid')
+            ->select('j.judgingid', 'j.submitid', 'j.judgehost_name', 'j.result',
+                'AVG(jr.runtime) AS runtime_avg', 'COUNT(jr.runtime) AS ntestcases',
+                '(j.endtime - j.starttime) AS duration'
+            )
+            ->andWhere('r.repeat_rejudgingid = :repeat_rejudgingid')
+            ->setParameter(':repeat_rejudgingid', $rejudging->getRepeatRejudgingId())
+            ->groupBy('j.judgingid')
+            ->orderBy('j.judgingid')
+            ->getQuery()
+            ->getResult();
+
+        $submissions = array();
+        $judgehosts = array();
```

Comment:
You might want to upgrade this method to shorthand array syntax while you're at it (but no necessity):
`$sumissions = [];`

Reply:
Done